src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -24,6 +24,10 @@ class Tribe__Events__REST__V1__Endpoints__Archive_Event
 		'status'      => 'post_status',
 		'post_parent' => 'post_parent',
 		'include'     => 'post__in',
+		'starts_before' => 'starts_before',
```

Comment:
Add argument documentation for these parameters in he `READ_args` method of this class.